DASH-IF-IOP.bs.md

```diff
@@ -10,9 +10,1003 @@ Live discussion in #document-authoring on Slack.
+
+#### Moving the period start point (indexed addressing) #### {#timing-addressing-indexed-startpoint}
+
+When splitting periods in two or performing other types of editorial timing adjustments, a service might want to start a period at a point after the "natural" start point of the [=representations=] within.
+
+For [=representations=] that use [=indexed addressing=], perform the following adjustments to set a new [=period=] start point:
+
+1. Update `SegmentBase@presentationTimeOffset` to indicate the desired start point on the [=sample timeline=].
+1. Update `Period@duration` to match the new duration.
+
+### Explicit addressing ### {#timing-addressing-explicit}
+
+A representation that uses <dfn>explicit addressing</dfn> consists of a set of [=media segments=] accessed via URLs constructed using a template defined in the MPD, with the exact time span covered by each [=media segment=] described in the MPD.
+
+Clauses in section only apply to [=representations=] that use explicit addressing.
+
+<figure>
+	<img src="Images/Timing/ExplicitAddressing.png" />
+	<figcaption>Explicit addressing uses a segment template that is combined with explicitly defined time spans for each [=media segment=] in order to reference [=media segments=].</figcaption>
+</figure>
+
+The MPD SHALL contain a `SegmentTemplate/SegmentTimeline` element that contains a set of [=segment references=] which satisfies the requirements defined in this document. The references exist as a sequence of `S` elements, each of which references one or more [=media segments=] with start time `S@t` and duration `S@d` in `SegmentTemplate@timescale` units on the [=sample timeline=]. The `SegmentTemplate@duration` attribute SHALL NOT be present.
+
+To enable concise reference definitions, an `S` element may be a repeating [=segment reference=] that indicates a number of repeated consecutive [=media segments=] with the same duration. The value of `S@r` SHALL indicate the number of additional consecutive [=media segments=] that exist.
+
+Note: Only additional references are counted, so `S@r=5` indicates a total of 6 consecutive [=media segments=] with the same duration.
+
+The start time of a [=media segment=] SHALL be calculated from the start time and duration of the previous [=media segment=] if not specified by `S@t`. There SHALL NOT be any gaps or overlap between [=media segments=].
+
+The value of `S@r` SHALL be nonnegative, except for the last `S` element which MAY have a negative value in `S@r`, indicating that the repeated references continue indefinitely up to a [=media segment=] that either ends at or overlaps the period end point.
+
+[[#timing-mpd-updates|Updates to a dynamic MPD]] MAY add more `S` elements, remove expired `S` elements or increase the value of `S@r` on the last `S` element but SHALL NOT otherwise modify existing `S` elements.
```

Comment:
As a removed `S` element may contain an `S@t` and the following may omit it, do we need to explicitly allow the introduction of the omitted `S@t` allowed as long as the value does not change from the original computed one?